pymlx/model.py

```diff
@@ -7,12 +7,15 @@
 from numpy import float32
 from scipy.sparse import csr_matrix
 from xgboost import XGBModel
+from lightgbm import LGBMModel, Booster
 from .featurizer import Featurizer
 
 
 class BinaryClassifier:
     def __init__(self, predictor, featurizer):
-        assert isinstance(predictor, XGBModel) and isinstance(featurizer, Featurizer)
+        # XgBoost model or LightGbm model is accepted
+        assert (isinstance(predictor, XGBModel) or isinstance(predictor, LGBMModel)) and isinstance(featurizer, Featurizer)
```

Comment:
We shouldn't check this anymore. In the future, we may use something else, e.g. Keras, as long as it shares similar API.